go.mod

```diff
@@ -24,6 +25,8 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require github.com/stoewer/go-strcase v1.2.0 // indirect
```

Comment:
This should have ended up in the longer list of indirect dependencies below right?

Reply:
Not sure honestly. I'll make sure to run `go mod tidy` as that will put it in the right spot.